samples/net/download/README.rst

```diff
@@ -24,12 +24,12 @@ Overview
 
 The sample first initializes the :ref:`nrfxlib:nrf_modem` and AT communications.
 Next, it provisions a certificate to the modem using the :ref:`modem_key_mgmt` library if the :ref:`CONFIG_SAMPLE_SECURE_SOCKET <CONFIG_SAMPLE_SECURE_SOCKET>` option is set.
-The provisioning of the certificates must be done before connecting to the LTE network since the certificates can only be provisioned when the device is not connected.
+When compiling for an nRF91 series device, the provisioning of the certificates must be done before connecting to the LTE network since the certificates can only be provisioned when the device is not connected.
```

Comment:
```suggestion
When using an nRF91 Series device, the provisioning of the certificates must be done before connecting to the LTE network since the certificates can only be provisioned when the device is not connected.
```